content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 2022-09-27T10:00-07:00
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: true
+heroImage: blog/release-post/4.0/Release.png
+socialImage: blog/release-post/4.0/Release.png
+changelogItems:
+---
+
+WIP INTRO: Sourcegraph 4.0, the latest version of our code intelligence platform, is now available. Our code intelligence platform includes a dozen new features to grok code, spend more time in flow, and execute big decisive refactors across your codebase.
+
+A code intelligence platform indexes and understands all of your organization's code, no matter where it's stored or what language it's in. With Sourcegraph, devs and engineering leaders have a complete understanding of their codebase, so they can learn about code and operate more efficiently. 
+
+Read on to discover everything that shipped in Sourcegraph 4.0.  
+
+### 🧠 Code intelligence: uplevel your code search
+
+<br />
+<Badge text="Code Insights" link="/code-insights" color="green" size="small" />
+
+#### Understand usage and search structure with Code Insights within the search UI
```

Comment:
```suggestion
#### Understand usage and code structure with Code Insights in the search UI
```

Reply:
(this is more correct, we don't do anything about search structure so it'd be incorrect to say that)

Reply:
Thanks - committing it now.